src/Cardano/Wallet/Primitive.hs

```diff
@@ -241,7 +242,8 @@ instance Dom UTxO where
 
 balance :: UTxO -> Integer
 balance =
-    Map.foldl' (\total out -> total + fromIntegral (getCoin (coin out))) 0 . getUTxO
+    Map.foldl' (\total out -> total + fromIntegral (getCoin (coin out))) 0
+        . getUTxO
```

Comment:
In such case, using an intermediate variable makes more sense to me.

```hs
balance =
    Map.foldl' fn 0 . getUTxO
  where
    fn :: Integer -> TxOut -> Integer
    fn total out = total + fromIntegral (getCoin (coin out))
```

Reply:
Thanks. Updated to match your suggestion.